CenterForOpenScience/ember-osf-web
# [ENG-2629] Upgrade to FontAwesome 5

- Ticket: [ENG-2629](https://openscience.atlassian.net/browse/ENG-2629)
- Feature flag: n/a

## Purpose

Upgrade from FontAwesome 4 to FontAwesome 5

## Summary of Changes

1. Change the addon we use for FontAwesome
2. Update outdated icon names with new icon names
3. Update the font collection as necessary
4. Replace non-FaIcon uses of icons with using the component
5. Update css for `<i>` to apply to `<svg>`
6. Update tests
7. Use default controls for Institutional Carousel

<img width="1070" alt="Screen Shot 2021-03-02 at 9 49 57 AM" src="https://user-images.githubusercontent.com/6599111/109674633-1332fa80-7b45-11eb-943d-15bead0ee6aa.png">


## Side Effects

Yeah, it's entirely possible I've missed some icons. The result will be blank spaces where icons should be. Also, some icons will have a different look.

## QA Notes

On the one hand, this is risky because it's hard to see uses of icons everywhere in the codebase, and this is a wide-spreading change. On the other hand, missing an icon here and there isn't the most critical of bugs. But that's the most likely case of problems, is that an icon will be missing where there should be an icon.
